reckless_lib/src/url.rs

```diff
@@ -0,0 +1,92 @@
+use std::fmt;
+
+use crate::utils::get_reckless_dir;
+
+/// This struct will make sure our URL's are of the
+/// correct format
+#[derive(Clone)]
+pub struct URL {
+    // the url string
```

Comment:
```suggestion
    /// the url string
```

in this way will be a comment

Reply:
Thanks, just fixed.